homeassistant/components/conversation/__init__.py

```diff
@@ -69,7 +69,7 @@
 def agent_id_validator(value: Any) -> str:
     """Validate agent ID."""
     hass = core.async_get_hass()
-    manager = _get_agent_manager(hass)
+    manager = get_agent_manager(hass)
```

Comment:
We shouldn't expose this. This is an internal way the integration is managing this. We also plan on updating this to be actual service entities like `stt` and `tts`, but are not there yet.

Reply:
Thank you. Looking forward to the updates.